src/main/java/org/takes/facets/fork/FkHitRefresh.java

```diff
@@ -86,14 +85,11 @@ public FkHitRefresh(final File file, final List<String> cmd,
         final Take tke) {
         this(
             file,
-            new Runnable() {
-                @Override
-                public void run() {
-                    try {
-                        new ProcessBuilder().command(cmd).start();
-                    } catch (final IOException ex) {
-                        throw new IllegalStateException(ex);
-                    }
+            () -> {
```

Comment:
@Iprogrammerr I appreciated this change, but it didn't remove any nulls.

Reply:
> @Iprogrammerr I appreciated this change, but it didn't remove any nulls.

This did not take any additional effort and will be done anyway #884.